lib/DxilContainer/DxilPipelineStateValidation.cpp

```diff
@@ -20,6 +20,7 @@
 #include "llvm/IR/DataLayout.h"
 #include "llvm/IR/Module.h"
 #include "llvm/Support/raw_ostream.h"
+#include <assert.h> // Needed for DxilPipelineStateValidation.h
```

Comment:
clang-format autopublish moved this assert here, but it needs to be included before `DxilPipelineStateValidation.h`.  So I believe it needs to be separated by a blank line, right?

Reply:
@tex3d  Yes, you're right. Added a blank line to fix it.